Keyboards/KeyboardsBase/KeyboardStyling.swift

```diff
@@ -40,7 +40,9 @@ var keysThatAreSlightlyLarger = [
   "chevron.right",
   "shift",
   "shift.fill",
+  "capslock",
```

Comment:
Again the dedication here is amazing. Looking into and finding `keysThatAreSlightlyLarger` is great, but also has me worried about the state of some of the code that we're using so many random variables everywhere.

A basic ask would be if you had ideas for how we can improve the code base, it'd be great if you could open some issues. Anything that you've seen that might be some solid refactoring would be very welcome!

Reply:
Good point 🤔 I'll do a quick sweep over the codebase and open a few issues for code quality ^^

Reply:
Thanks so much, @damien-rivet! Always looking for some `good first issues` for new folks to work on 😊